README.md

```diff
@@ -1731,6 +1732,14 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 - [UVDesk](https://www.uvdesk.com/) - UVDesk community is a service oriented, event driven extensible opensource helpdesk system that can be used by your organization to provide efficient support to your clients effortlessly whichever way you imagine. ([Demo](https://demo.uvdesk.com/), [Source Code](https://github.com/uvdesk/community-skeleton)) `MIT` `PHP`
 - [ZenTao](https://www.zentao.pm/) - An agile(scrum) project management system/tool. ([Demo](https://demo.zentao.pm/user-login.html), [Source Code](https://github.com/easysoft/zentaopms)) `ZPL-1.2` `PHP`
 
+### Software Development - Security & QA
+
+**[`^        back to top        ^`](#)**
+
+- [Mega-Linter](https://megalinter.github.io/latest) - Mega-Linter can handle any type of project thanks to its 70+ embedded Linters, its advanced reporting, runnable on any CI system or locally, with assisted installation and configuration, able to apply formatting and fixes. `AGPL-3.0` `Python`
+- [Scanmycode CE (Community Edtion)](https://www.scanmycode.today/) - Code Scanning/SAST/Static Analysis/Linting using many tools/Scanners with One Report - Scanmycode Community Edition (CE). ([Source Code](https://github.com/marcinguy/scanmycode-ce)) `BSD-3-Clause LGPL-2.1 Commonsclause` `Python`
+- [SonarQube](https://www.sonarqube.org)SonarQube is an open platform to manage code quality. `LGPL-3.0` `Java`
```

Comment:
- you should add a link to the source code: https://github.com/SonarSource/sonarqube
- misgging ` - ` between `)SonarQube`

Reply:
Fixed